### PR TITLE
created calendar

### PR DESCRIPTION
**Pull request by mykola-chernysh**
[DEMO LINK](https://mykola-chernysh.github.io/layout_calendar/)
[TEST REPORT LINK](https://mykola-chernysh.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/calendar.scss [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+  &__day {
+    @include square($cell-size);
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $color-bg-day;
+    border: 1px solid $color-border-day;
+    font-family: Arial, Helvetica, sans-serif;
```
Comment (alexandra-protyanova) [inline src/styles/blocks/calendar.scss]: move to `html`

File: src/styles/blocks/reset.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+ul,
+ol,
+li {
+  list-style: none;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
```
Comment (alexandra-protyanova) [inline src/styles/blocks/reset.scss]: use `inherit` in child tags, now u have 2 different font-families

File: src/styles/blocks/calendar.scss [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    }
+
+    @for $number-day from 1 through 31 {
+      &:nth-child(#{$number-day})::before {
+        content: "#{$number-day}";
+      }
+    }
+  }
+
+  @for $month-length from 28 through 31 {
```
Comment (alexandra-protyanova) [inline src/styles/blocks/calendar.scss:36]: move loops to different file

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

File: src/styles/blocks/calendar.scss (hunk 3 above)
Comment (mykola-chernysh) [reply src/styles/blocks/calendar.scss:36]: move to another file as variables?

**Review by Bohdan-Maksymiuk — APPROVED** [review]

**Pull request closed without merging**